composer.lock

```diff
@@ -961,7 +961,7 @@
     "prefer-stable": false,
     "prefer-lowest": false,
     "platform": {
-        "php": "^7.2"
```

Comment:
Docker will remove the need to downgrade PHP for local-machine reasons, so please revert this. If you need php 7.0 until the Docker work is done, the specification of php 7.0 can be kept local to your machine.

Reply:
Reverted